charts/aws-ebs-csi-driver/values.yaml

```diff
@@ -126,6 +126,9 @@ controller:
   #    topologyKey: kubernetes.io/hostname
   #    whenUnsatisfiable: ScheduleAnyway
   topologySpreadConstraints: []
+  # The default filesystem type of the volume to provision when fstype is unspecified in the StorageClass.
+  # If the default is not set and fstype is unset in the StorageClass, then no fstype will be set
+  defaultFsType: ext4
```

Comment:
Please move this to the correct alphabetical place in the list of values under controller.

Reply:
Done